manifests/baremetal/keepalived.yaml

```diff
@@ -56,14 +56,11 @@ spec:
       privileged: true
     image: {{ .Images.KeepalivedBootstrap }}
     command:
-    - /usr/sbin/keepalived
-    args:
-    - "-f"
-    - "/etc/keepalived/keepalived.conf"
-    - "--dont-fork"
-    - "--vrrp"
-    - "--log-detail"
-    - "--log-console"
+    - /bin/sh
+    - -c
+    - |
+       export NSS_SDB_USE_CACHE=no
```

Comment:
Would it be simpler to
```
    env:
    - name: NSS_SDB_USE_CACHE
      value: "no"
```
?

Reply:
Indeed. Please try that, @yboaron

Reply:
@cgwalters , Yep, looks like a nicer way to set this variable. I'll verify it on my env and will update the PR. 10x